src/Diagnostics.DataProviders/KeyVaultCertLoader/KeyVaultCertLoaderBase.cs

```diff
@@ -1,32 +1,42 @@
 ﻿using System;
 using System.Security.Cryptography.X509Certificates;
-using Microsoft.Azure.KeyVault;
-using Microsoft.Azure.Services.AppAuthentication;
-
+using Diagnostics.Logger;
 namespace Diagnostics.DataProviders.KeyVaultCertLoader
 {
     public abstract class KeyVaultCertLoaderBase
     {
-        protected abstract string KeyVault { get; set; }
-
-        protected abstract string CertificateName { get; set; }
-
+        protected abstract string Thumbprint { get; set; }
         public X509Certificate2 Cert { get; private set; }
 
-        public async void LoadCertFromKeyVault()
+        public async void LoadCertFromAppService()
```

Comment:
Using `async void` means callers can't `await` this to get exception details. There is some code outside of `try/catch` in the method body, so this has a possibility of losing an exception or crashing the service.

Reply:
@dagrooms52 `async` was previously there when we were loading from Key Vault. I missed removing that when I made the change to load from App Service. Should be updated now